refinery/tool_manager/models.py

```diff
@@ -400,6 +416,33 @@ def update_file_relationships_with_urls(self):
             )
         self.set_tool_launch_config(tool_launch_config)
 
+    def is_running(self):
+        if self.is_workflow():
+            return True if self.analysis.running() else False
```

Comment:
Don't need the bracketing if-else. (Or use `bool()` if the point really is to cast.)